package.json

```diff
@@ -6459,4 +6879,4 @@
 		"lodash": "^4.17.21",
 		"path-browserify": "^1.0.1"
 	}
-}
\ No newline at end of file
+}
```

Comment:
Remove the extra change. Is this coming from the `npm run sort-bindings`? If it is, I will take a look and fix it.

Reply:
Yes, that came from `npm run sort-bindings`.